packages/api/app/Models/Traits/Params.js

```diff
@@ -75,6 +76,7 @@ class Params {
 				'created_at',
 				'updated_at',
 			],
+			technology_orders: ['id', 'unit_value', 'user_id', 'created_at', 'updated_at'],
```

Comment:
Precisamos declarar o status e outros filtros aqui? 👀 
```suggestion
			technology_orders: ['id', 'status', 'unit_value', 'user_id', 'created_at', 'updated_at'],
```